### DISCUSSION TITLE
Alienware M18 R2 (2024)

### DISCUSSION THREAD

**Discussion by NextGenY** [T-Troll/alienfx-tools#468] [Q&A]
Hello, I'm trying to use the AlienFX controls to "upgrade" the visuals for my keyboard. Unfortunately, it seems that AlienFX is only detecting my escape key? That's the only key I'm able to change the colour with.

I've searched a bit for some answers but can't find anything that stands out. Does anyone have any suggestions on what I can try?

**Comment by T-Troll** [comment]
First, you need DEFINE the lights at Devices and Grids tab.
Then, you can CONTROL it (other lights-related tabs).
Read beginner's guide.
